src/components/organisms/ErrorBoundary/ErrorBoundary.tsx

```
@@ -1,108 +1,137 @@
+          <Text 
+            style={styles.title}
+            accessibilityRole="header"
+            accessibilityLabel="Error title: Something went wrong"
+          >
+            Something went wrong
+          </Text>
+          <Text
+            style={styles.message}
+            accessibilityLabel="Error description"
+          >
+            We're sorry, but an unexpected error occurred.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)